Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.kt

```diff
@@ -207,12 +208,20 @@ class MainActivityNavigator(
     override fun showColorPickerDialog() {
         if (findFragmentByTag(COLOR_PICKER_DIALOG_TAG) == null) {
             toolReference.tool?.let {
+                val eraserColor: Int
+                if (toolReference.tool?.toolType == ToolType.ERASER) {
+                    eraserColor = Color.TRANSPARENT
+                    it.drawPaint.color = eraserColor
+                }
                 val dialog = ColorPickerDialog.newInstance(
                     it.drawPaint.color,
                     mainActivity.model.isOpenedFromCatroid,
                     mainActivity.model.isOpenedFromFormulaEditorInCatroid,
                     mainActivity.model.colorHistory
                 )
+                if (toolReference.tool?.toolType != ToolType.ERASER) {
```

Comment:
This if seems obsolete since this method is never called if the Erasertool is in use.

Reply:
I removed the obsolete changes in the function.